eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -147,31 +149,102 @@ private void build(final File project) throws IOException {
                 target
             );
         }
-        Logger.info(this, "Building rust project..");
-        try (
-            VerboseProcess proc = new VerboseProcess(
-                new ProcessBuilder("cargo", "build")
-                    .directory(project)
-            )
-        ) {
-            proc.stdout();
-        } catch (final IllegalArgumentException exc) {
-            throw new BuildFailureException(
+        if (BinarizeMojo.sameProject(
+            project.toPath(), this.cache
+            .resolve("Lib")
+            .resolve(project.getName())
```

Comment:
@levBagryansky Pretty strange formatting for me. Can you please move `this.cache` to the new line?